packages/ps/src/commands/ps/wait.ts

```diff
@@ -0,0 +1,80 @@
+import {Command, flags} from '@heroku-cli/command'
+import * as Heroku from '@heroku-cli/schema'
+import {cli} from 'cli-ux'
+
+async function delay(millis: number) {
+  return new Promise(resolve => setTimeout(resolve, millis))
+}
+
+export default class Wait extends Command {
+  static description = 'wait for a release to cycle in'
+  static help = `
+When a release is created, it may take a while for all dynos to be
+running the new version. This is especially true for applications in
+Heroku Private Spaces or using the Common Runtime preboot feature,
+where dynos cycle in gradually when a new release is deployed. This
+command allows you to wait until all dynos are on the latest release
+version.
+  `
```

Comment:
Is `help` a thing? I followed the same format as `description` hoping that's it but didn't find any docs.

Reply:
@uhoh-itsmaciek I'm not aware of `help` in this context.  I've always set all my command information on the `description` field.   I would be tempted to do the same here e.g. just combine everything in `description`.  @RasPhilCo might have some other opinions.